js/cards.js

```diff
@@ -70,3 +84,42 @@ export function renderCards(cards, htmlLocation, replace = false) {
     );
   }
 }
+
+function getOwnedCards(cards_data) {
+  let ownedCards = [];
+  let storage = { ...localStorage };
+  for (let item in storage) {
+    if (item.slice(0, 4) === "card") {
+      let card_id = item.split("-")[1];
+      ownedCards.push(
+        cards_data.find((card) => card["Collector Number"] === card_id)
+      );
```

Comment:
I wonder how the performance of this will hold up when we have hundreds of cards. Maybe it'd be worth having a `cards_by_collector_number` object alongside the rarity one. That is a memory tradeoff though.

Reply:
That's a good idea, let me write a reminder to do that if performance starts to suffer. About pushing it all to localStorage, I feel like it's a little overkill given that we already have all the info in cards_data, cards_by_rarity and potentially cards_by_collector_number, each of them using references to the same card objects, while pushing it to localStorage would bring another array but with brand new references not shared with the others (localStorage saves strings only, so the objects would have to be rebuilt on retrieval). What we can do is store preprocessed data in localStorage to save time. I already made a commit to save card count by rarity in localStorage and keep track of pulls, we can add more things to it if, again, performance starts to suffer.

Reply:
Also took away the "darkened card" overlay. It's gone now, poof.